shoop_tests/core/test_customers.py

```diff
@@ -63,3 +60,22 @@ def test_customer_tax_group2():
 def test_customer_tax_group3(rf, admin_user):
     # SHOOP-1882
     assert type(CustomerTaxGroup.get_default_company_group().__str__()) == str
+
+
+def default_group_basic_test(contact_cls, create_contact):
+    new_contact = create_contact()
+    assert new_contact.groups.count() == 1
+    default_group = new_contact.groups.first()
+    assert type(CustomerTaxGroup.get_default_company_group().__str__()) == str
+    assert default_group == new_contact.get_default_contact_group()
+    assert default_group.identifier == contact_cls.default_contact_group_identifier
+
+    some_other_contact = create_contact()
+    assert some_other_contact.groups.count() == 1
+
+
+@pytest.mark.django_db
+def test_default_groups():
+    default_group_basic_test(AnonymousContact, AnonymousContact)
```

Comment:
This would be better phrased as a parametrization of the single test, not a helper that runs three tests with parameters.

See [`pytest.parametrize`](https://pytest.org/latest/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions).